app/src/main/AndroidManifest.xml

```diff
@@ -21,6 +21,17 @@
         android:supportsRtl="true"
         android:theme="@style/AppTheme"
         tools:replace="android:allowBackup">
+
+        <provider
```

Comment:
Can you add a link to Android documentation used to implement this change? (In case we need to do changes related to this in the future)

Reply:
Sure.
Please refer to
https://developer.android.com/reference/android/support/v4/content/FileProvider.html
https://inthecheesefactory.com/blog/how-to-share-access-to-file-with-fileprovider-on-android-nougat/en

Reply:
Thanks. Can you add it to the code?

Reply:
I have done that. Kindly check it.